docker-contract/docker-compose.yml

```diff
@@ -5,9 +5,9 @@ networks:
     name: mojaloop-pisp-net
 
 services:
-  mojaloop-testing-toolkit:
```

Comment:
Was there any specific reasoning behind changing the names Lewis?

Reply:
In  mojaloop-testing-toolkit  repo they used mojaloop instead of ml, so used same service name here but that's ok to rename it.

Reply:
Just to save some typing and keep names consistent with the other containers in this file

Reply:
ok.